CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ execute_process(COMMAND python ${podio_DIR}/../../../python/podio_class_generato
 #--- Declare ROOT dependency ---------------------------------------------------
 list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS})
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-find_package(ROOT REQUIRED COMPONENTS RIO Tree Physics)
+find_package(ROOT REQUIRED COMPONENTS RIO Tree Physics ROOTDataFrame)
```

Comment:
I guess this ROOTDataFrame only exists in newer versions of root.
So it might be better to not include this here and thus not force this on everyone

Reply:
Ok, I'll add a cmake  option for building `dataframe` subdirectory.